skema/program_analysis/CAST/matlab/matlab_to_cast.py

```diff
@@ -78,6 +82,7 @@ def generate_cast(self) -> List[CAST]:
         """Interface for generating CAST."""
 
         # remove comments from tree before processing
+
         modules = self.run(remove_comments(self.tree.root_node))
         return [CAST([module], "matlab") for module in modules]
```

Comment:
I'm going to guess that this line would be a good place to start for resolving the extra list being wrapped around the CAST object.

Reply:
I think this is based on the Fortran TS2CAST which can return multiple CAST modules for a single file. This is then taken into account in multi_file_ingester.py to handle creating Gromet for each module.

Reply:
If possible I would like to shift this fix to the next PR